test/integration/targets/nxos_install_os/tasks/main.yaml

```diff
@@ -1,12 +1,12 @@
 ---
 # Upgrade using SSH
 - include: network_cli.yaml
-  when: connection_type == 'network_cli'
+  when: ansible_connection == 'network_cli'
 - include: network_local.yaml
-  when: connection_type == 'cli_local'
+  when: ansible_connection == 'cli_local'
```

Comment:
Either we need to have one entrypoint for local connection tasks or drop cli local connection tests as we did for the other modules.

Reply:
This block of code can be removed and delete the file tasks/network_local.yaml
```
- include: network_local.yaml
  when: ansible_connection == 'cli_local'
```